inference-engine/include/ie_blob.h

```diff
@@ -756,6 +827,30 @@ class TBlob : public MemoryBlob {
     void* getHandle() const noexcept override {
         return _handle;
     }
+
+    /**
+     * @brief Creates an new empty rvalue LockedMemory object.
+     *
+     * @return rvalue for the empty locked object of type T
+     */
+    virtual LockedMemory<T> data() const noexcept {
+        return std::move(lockme<T>());
+    }
+
+    /**
+     * @brief Clones Blob object of the particular class. Clone references the same memory as this object.
+     *
+     * @return Pointer to the cloned object
+     */
+    Blob* clone() const override {
+        TBlob* cloned = new TBlob<T>(this->tensorDesc, this->data(), this->size());
```

Comment:
What GitHub did with my previous comment?

Using `this->data()` here is dangerous, since it will lead to `map/unmap` calls from allocator. The `clone` method should copy `_handle` and `_allocator` fields directly.

Reply:
Existing pre-allocated memory at `_handle` is utilized with dedicated allocator now.